Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/utils/FileBrowserHelper.java

```diff
@@ -30,7 +30,7 @@
 public final class FileBrowserHelper
 {
   public static final HashSet<String> GAME_EXTENSIONS = new HashSet<>(Arrays.asList(
-          "gcm", "tgc", "iso", "ciso", "gcz", "wbfs", "wia", "rvz", "wad", "dol", "elf"));
+          "gcm", "tgc", "bin", "iso", "ciso", "gcz", "wbfs", "wia", "rvz", "wad", "dol", "elf"));
```

Comment:
Should we really be supporting `.bin`? This is an already super generic extension that's been overloaded into oblivion for dumps

Reply:
Many triforce games are in .bin formats by default, which is why I added it